docs.package.json

```diff
@@ -36,8 +36,8 @@
     "label": "testbed"
   },
   {
-    "repo": "osism/cloud-in-a-box",
-    "source": "doc",
+    "repo": "osism/docs",
```

Comment:
This is the new repository: https://github.com/osism/osism.github.io

Reply:
For testbed: Should we use osism/testbed/doc OR better osism/osism.github.io/docs/advanced-guides/testbed.md?

Reply:
Ok, we are consuming the osism.github.io pages now.